website/docs/source/v2/virtualbox/configuration.html.md

```diff
@@ -54,6 +54,14 @@ config.vm.provider "virtualbox" do |v|
 end
 ```
 
+To have backward compatibility:
```

Comment:
@kikitux good idea, but I think we want to encourage `Vagrant.require_version` instead.